relay-server/src/actors/store.rs

```diff
@@ -1030,14 +988,28 @@ impl KafkaMessage {
     }
 }
 
-/// Message sent to the StoreForwarder containing an event
+/// Message sent to the [`StoreForwarder`] containing an event.
 #[derive(Clone, Debug)]
 pub struct StoreEnvelope {
     pub envelope: Envelope,
     pub start_time: Instant,
     pub scoping: Scoping,
 }
 
+/// Envelope for the possible Messages that can be sent to the [`StoreForwarder`].
+pub enum StoreMessageEnvelope {
```

Comment:
Same as above, this should be called `StoreEnvelope` as per convention. To avoid the naming clash, we should be calling the other one `ProduceEnvelope`, which describes better what it actually does.

```suggestion
pub enum StoreEnvelope {
```